scripts/post-promote-tasks-job.ts

```diff
@@ -9,29 +9,49 @@ const {head, base} = yargs(process.argv.slice(2))
   })
   .parseSync();
 
-interface TaskConfig {
-  [channel: string]: string;
-}
-
-//TODO(estherkim): add release tagger tasks
-const PUBLISH_NPM: TaskConfig = {
+const PUBLISH_NPM: {[channel: string]: string} = {
   nightly: 'nightly',
   stable: 'latest',
 };
 
-const RELEASE_CALENDAR: TaskConfig = {
+const RELEASE_CALENDAR: {[channel: string]: string} = {
   nightly: 'nightly',
   'beta-traffic': 'beta',
   stable: 'stable',
   lts: 'lts',
 };
 
+const RELEASE_TAGGER: {[channel: string]: Record<string, string>} = {
+  'beta-opt-in': {
+    headChannel: 'beta-opt-in',
+    baseChannel: 'stable',
+  },
+  'beta-traffic': {
+    headChannel: 'beta-percent',
+    baseChannel: 'stable',
+  },
+  stable: {
+    headChannel: 'stable',
+    baseChannel: 'stable',
+  },
+  lts: {
+    headChannel: 'lts',
+    baseChannel: 'lts',
+  },
+};
+
 async function setOutput() {
   const versionDiff = await getVersionDiff(head, base);
   const npm: {'amp-version': string; tag: string}[] = [];
   const calendar: {'amp-version': string; channel: string}[] = [];
+  const tagger: {
+    action: string;
+    head: string;
+    base: string;
+    channel: string;
+  }[] = [];
 
-  versionDiff.forEach(({channel, version}) => {
+  for await (const {channel, version} of versionDiff) {
```

Comment:
Why does this need to be a `for await`? You await on `getVersionDiff(head, base)` eariler, so `versionDiff` should be a concrete value, not a Promise